cd2h_repo_project/config.py

```diff
@@ -88,10 +88,9 @@ def _(x):
 # Email configuration
 # ===================
 #: Email address used to send emails.
-# TODO: Create a project email once name is settled on
-SUPPORT_EMAIL = ""
+SUPPORT_EMAIL = ""  # TODO::Hoster: CONFIGURE ME
 #: Disable email sending by default.
-MAIL_SUPPRESS_SEND = True
+MAIL_SUPPRESS_SEND = True  # TODO::Hoster: Set to False
```

Comment:
> What would happen if this is set to False without SUPPORT_EMAIL set, and a contact form is submitted?
There will be an error thrown most likely. Note that sending emails via northwestern only works on stage/production because these servers are allowed - our local machines are not (except when we are wired in to the network most probably).

> How about set it to True or False based on whether SUPPORT_EMAIL is set?
Explained out aloud.